Userland/Libraries/LibWeb/CSS/CSSStyleSheet.cpp

```diff
@@ -51,6 +51,7 @@ void CSSStyleSheet::visit_edges(Cell::Visitor& visitor)
     visitor.visit(m_style_sheet_list.ptr());
     visitor.visit(m_rules);
     visitor.visit(m_owner_css_rule);
+    visitor.visit(m_default_namespace_rule);
```

Comment:
Sorry! I thought this would be covered by it already being inside `m_rules`?

Reply:
Technically yes! But let's be safe boys and not rely on transitive visits :)

Reply:
:+1: